src/components/shared/Entity/WrapperWithHeader.tsx

```diff
@@ -1,21 +1,27 @@
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 import { Accordion, AccordionDetails, AccordionSummary } from '@mui/material';
-import { ReactNode, useState } from 'react';
+import { ReactNode, useEffect, useState } from 'react';
 
 interface Props {
     header: ReactNode;
     children: ReactNode;
-    defaultExpand?: boolean;
+    forceClose?: boolean;
 }
 
-function WrapperWithHeader({ header, children, defaultExpand }: Props) {
-    const [expanded, setExpanded] = useState(defaultExpand ?? true);
+function WrapperWithHeader({ header, children, forceClose }: Props) {
+    const [expanded, setExpanded] = useState(true);
     const handlers = {
         change: () => {
             setExpanded(!expanded);
         },
     };
 
+    useEffect(() => {
+        if (forceClose) {
+            setExpanded(false);
+        }
+    }, [forceClose]);
```

Comment:
Allow a sort of "flag" that will let the parent collapse this component if it needs.